### PR TITLE
[13.0][IMP] account_invoice_mgmt_dms: merge compression + document validation

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
add merge compression

ghostscript

### CODE REVIEW

**Comment by dalonsod** [comment]
:+1: please update `README.rst/index.html` files

File: account_invoice_mgmt_dms/models/stock_move.py
Comment (dalonsod) [inline account_invoice_mgmt_dms/models/stock_move.py]: Was this a previous bug that is fixed here? If I'm right, please separate it in a previous and standalone commit

File: account_invoice_mgmt_dms/models/dms_file.py
Comment (dalonsod) [inline account_invoice_mgmt_dms/models/dms_file.py]: If we use `raise_if_not_found=True`, `directory_id` could be empty, so an `if` statement previous `for` statement could be added, and `for` skipped if directory is not found.

Anyway, carrier doc directory XML Id should be always accessed, because we cannot delete it (https://github.com/solvosci/slv-account/blob/5fa2f26118ff32be57508c95178674060c56c9c9/account_invoice_mgmt_dms/models/dms_directory.py#L11-L28)

File: account_invoice_mgmt_dms/models/dms_file.py
Comment (dalonsod) [inline account_invoice_mgmt_dms/models/dms_file.py]: Please check if every user that can delete a file have read and write access to `stock.move` model (e.g. a DMS user). if not, please use `sudo()`.

File: account_invoice_mgmt_dms/migrations/13.0.1.9.0/post-migration.py
Comment (dalonsod) [inline account_invoice_mgmt_dms/migrations/13.0.1.9.0/post-migration.py]: Using a subselect could be expensive. Not sure if this alternative will work, but could you check?
```suggestion
        f"""
        UPDATE
	        stock_move
        SET
	        with_doc = true
        FROM
	        stock_move sm
        INNER JOIN
	        stock_picking sp on sp.id=sm.picking_id
        INNER JOIN
	        dms_file df on df.proceeding=sp.name
	        and df.directory_id = {directory_id.id}
        WHERE
	        stock_move.id=sm.id
        ;""",
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments

**Comment by ChristianSantamaria** [comment]
Done :+1: 
I'm going to leave this PR separate for ghostscript
New pr #68

File: account_invoice_mgmt_dms/views/account_move_views.xml
Comment (dalonsod) [inline account_invoice_mgmt_dms/views/account_move_views.xml]: Remove unneeded space between attribute name and value. And, also, context variable name should be more specific, as context is a very general variable container for Odoo:
```suggestion
                    context="{'create_dms_file_document_validate': True}"
```

File: account_invoice_mgmt_dms/models/account_move.py
Comment (dalonsod) [inline account_invoice_mgmt_dms/models/account_move.py]: According to previous comment, required change:
```suggestion
        validate = self.env.context.get('create_dms_file_document_validate', False)
```
And proper code formatting

File: account_invoice_mgmt_dms/models/account_move.py
Comment (dalonsod) [inline account_invoice_mgmt_dms/models/account_move.py:167]: Is there are link in internet for `/ebook` setting addition, that can be used as explanation?

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments. It looks good anyway

File: account_invoice_mgmt_dms/README.rst
Comment (dalonsod) [inline account_invoice_mgmt_dms/README.rst]: README stuff seems to be generated with a wrong target version (15.0), please fix it @ChristianSantamaria

**Comment by DantePereyra** [comment]
Changes done.

**Comment by dalonsod** [comment]
@ChristianSantamaria could you add @DantePereyra as contributor of this addon. Once done, I'll merge it!

**Review by dalonsod — APPROVED** [review]
Let's merge!

**Pull request merged**
